src/inputs/ObjectSearch.tsx

```diff
@@ -190,12 +190,13 @@ class ObjectSearch extends Component<IObjectSearchProps> {
   }
 
   private renderOption (option: IEndpointOption) {
-    const { renderOption, renderSelected } = this.fieldConfig
+    const { isDisabled, renderOption, renderSelected } = this.fieldConfig
        , className = `${CX_PREFIX_SEARCH_CREATE}-item`;
 
     return (
       <Antd.Select.Option
         className={className}
+        disabled={isDisabled ? isDisabled(option) : false}
```

Comment:
I would make it `isOptionDisabled` and instead of adding it to the fieldConfig, add it to the component props, then your fieldConfig can contain `{ editProps: { isOptionDisabled } }` and we don't need to change the fieldConfig definition.